test/web/10-VerifiableCredentialStore.js

```diff
@@ -107,40 +113,47 @@ describe('VerifiableCredentialStore', () => {
     should.not.exist(credential);
   });
 
-  it('should query for an AlumniCredential with any issuer', async () => {
-    const hub = await createEdv({mock});
-    const vcStore = new VerifiableCredentialStore({hub});
+  it.skip('should query for an AlumniCredential with any issuer', async () => {
+    const hub = await mock.createEdv({keyResolver});
+
+    const vcStore = new VerifiableCredentialStore({edv: hub, invocationSigner});
 
     await vcStore.insert({credential: AlumniCredential});
-    // TODO: Change code to use spread operator when bedrock-karma is fixed
+
     const newCred = Object.assign({}, AlumniCredential, {id: 'foo'});
```

Comment:
I see a todo is being removed about replacing Object.assign() with a spread operator.  Is it still a problem to use spread here?

Reply:
Yes, spread operator still doesn't work. I get `unexpected token` error when using spread operator. I think maybe we need to update karma.js in bedrock-karma https://github.com/digitalbazaar/bedrock-karma/blob/master/package.json#L37 to latest since the one we are using is very old.